YesterdayApi/appsettings.json

```diff
@@ -1,4 +1,7 @@
 {
+  "ConnectionStrings": {
+    "yesterday": "Data Source=127.0.0.1;Initial Catalog=yesterday;persist security info=True;user id=sa;password=zaq1@WSX"
```

Comment:
A better way for the connection string:
"Server=localhost, 1433; Database=DBNAME; MultipleActiveResultSets=true; User Id=LOGIN; Password=PASSWORD;"